plugins/poll/assets/javascripts/views/poll.js.es6

```diff
@@ -3,17 +3,12 @@ import { on } from "ember-addons/ember-computed-decorators";
 export default Em.View.extend({
   templateName: "poll",
   classNames: ["poll"],
-  attributeBindings: ["data-poll-type", "data-poll-name", "data-poll-status"],
+  attributeBindings: ["data-poll-type", "data-poll-name", "data-poll-status", "data-poll-public"],
 
   poll: Em.computed.alias("controller.poll"),
 
   "data-poll-type": Em.computed.alias("poll.type"),
   "data-poll-name": Em.computed.alias("poll.name"),
   "data-poll-status": Em.computed.alias("poll.status"),
-
-  @on("didInsertElement")
```

Comment:
@ZogStriP Btw I'm not sure why this was required and the commit goes all the way back to 2014. I'm removing it because it is causing problems on mobile and doesn't seem to be used on desktop at all.